kubetest/e2e.go

```diff
@@ -58,12 +58,19 @@ func run(deploy deployer, o options) error {
 
 	dump := o.dump
 	if dump != "" {
-		if !filepath.IsAbs(dump) { // Directory may change
-			wd, err := os.Getwd()
-			if err != nil {
-				return fmt.Errorf("failed to os.Getwd(): %v", err)
-			}
-			dump = filepath.Join(wd, dump)
+		if absDump, err := filepath.Abs(dump); err != nil {
```

Comment:
There might be some clever reason that this used to just reimplement `filepath.Abs`; if anyone knows what it is and it's important, this should be changed back and the reimplementation presumably copied.